src/utils/redirects.test.js

```diff
@@ -57,18 +57,17 @@ describe('getRedirect', () => {
       expect(rLocation).toEqual(target)
       expect(rCode).toEqual(code)
 
-      // Detect redirect loops
+      // Detect redirect loops.
       const secondUrl = url.parse(addHost(rLocation))
       const secondRedirect = getRedirect(secondUrl.hostname, secondUrl.pathname)
       expect(secondRedirect).toEqual([])
     })
   }
 
-  describe('host redirects', () => {
-    // remove the www (when already HTTPS)
+  describe('Subdomain-based redirects', () => {
```

Comment:
minor: I think it's a convention to keep it one phrase long and start with lower case

Reply:
Not sure but OK, using cammelCase convention now for `describe()` calls. Please check.